plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -193,14 +169,9 @@ export class SonataFlowService {
     definitionId: string;
     serviceUrl: string;
   }): Promise<boolean> {
-    try {
-      const urlToFetch = `${args.serviceUrl}/management/processes/${args.definitionId}`;
-      const response = await fetch(urlToFetch);
-      return response.ok;
-    } catch (error) {
-      this.logger.debug(`Error when pinging workflow service: ${error}`);
-    }
-    return false;
+    const urlToFetch = `${args.serviceUrl}/management/processes/${args.definitionId}`;
+    const response = await fetch(urlToFetch);
+    return response.ok;
   }
 
   public async updateInstanceInputData(args: {
```

Comment:
this whole function should be deleted, it used to be wrongly used by retrigger
waste of time to fix issue in this code

Reply:
we are trying to keep code consistent. removing this function should be done in a separate story